### PR TITLE
Normalizing ossec_config tag name

### LINKED ISSUE #1011: Configuration tags normalization to Wazuh standard - QA tests changes
### Description

As part of the usability improvements, one of them is to normalize the configuration tags used in the `ossec.conf` file to the Wazuh standard. This supposes a change in all the tags that make use of the word `ossec` and should be replaced with `wazuh`.

After analyzing the default configuration file, the following changes were identified: 

- The only tag that should be modified is `<ossec_config>`
- The example address in `<email_from> ` has the **ossecm@** username

This supposes changes in the next set of files:

```
	- deps/wazuh_testing/wazuh_testing/tools/configuration.py
	- tests/integration/README.md
	- tests/integration/test_fim/test_files/test_audit/data/ossec.conf
	- tests/integration/test_fim/test_files/test_stats_integrity_sync/data/template_wazuh_conf.conf
	- tests/legacy/test_sca/test_basic_usage/data/sca_files_test_suite.yml
	- tests/system/provisioning/agentless_cluster/roles/master-role/files/ossec.conf
	- tests/system/provisioning/agentless_cluster/roles/worker-role/files/ossec.conf
	- tests/system/provisioning/basic_cluster/roles/agent-role/files/ossec.conf
	- tests/system/provisioning/basic_cluster/roles/master-role/files/ossec.conf
	- tests/system/provisioning/basic_cluster/roles/worker-role/files/ossec.conf
	- tests/system/provisioning/enrollment_cluster/roles/agent-role/files/ossec.conf
	- tests/system/provisioning/enrollment_cluster/roles/master-role/files/ossec.conf
	- tests/system/provisioning/enrollment_cluster/roles/worker-role/files/ossec.conf
```

### PR DESCRIPTION
**Pull request by pereyra-m**
Hello team,

This PR updates the ossec.conf configuration tags to Wazuh standard.

Closes #1011 .

# Tests checks

- [x] Proven that tests **pass** when they have to pass
- [x] Proven that tests **fail** when they have to fail

### CODE REVIEW

**Comment by pereyra-m** [comment]
#### Tests

The following IT were performed by Jenkins (Tier 0):

- Manager FIM (CentOS): https://ci.wazuh.info/job/test_integration/1099/console
- Agent FIM (Ubuntu - CentOS/Win Agents): https://ci.wazuh.info/job/test_integration/1129/console
- Manager Cluster (CentOS): https://ci.wazuh.info/job/test_integration/1130/console
-  Manager Logtest (CentOS): https://ci.wazuh.info/job/test_integration/1132/console

File: tests/integration/test_fim/test_files/test_audit/data/ossec.conf [hunk 1]
```diff
@@ -291,4 +291,4 @@
     <disabled>yes</disabled>
   </cluster>
 
-</ossec_config>
\ No newline at end of file
+</wazuh_config>
```
Comment (palaciosjeremias) [inline tests/integration/test_fim/test_files/test_audit/data/ossec.conf:294]: This is legacy, but we should add the trailing EOL here.

File: tests/legacy/test_sca/test_basic_usage/data/sca_files_test_suite.yml [hunk 2]
```diff
@@ -147,7 +147,7 @@ checks:
      - f:/var/ossec/etc/ossec.conf,/var/ossec/etc/internal_options.conf -> r:^FAKE
 
  - id: 200403
-   title: INVALID -- Content not exist. One file does not exists 
+   title: INVALID -- Content not exist. One file does not exists
    condition: any
    rules:
      - f:/var/ossec/etc/ossec.conf,/var/ossec/etc/internal_options.conf,/var/ossec/etc/ossec.conf_FAKE1 -> r:^FAKE
```
Comment (palaciosjeremias) [inline tests/legacy/test_sca/test_basic_usage/data/sca_files_test_suite.yml:153]: Ditto

**Review by palaciosjeremias — APPROVED** [review]

File: tests/integration/test_fim/test_files/test_audit/data/ossec.conf (hunk 1 above)
Comment (pereyra-m) [reply tests/integration/test_fim/test_files/test_audit/data/ossec.conf:294]: Done, thank you!

File: tests/legacy/test_sca/test_basic_usage/data/sca_files_test_suite.yml (hunk 2 above)
Comment (pereyra-m) [reply tests/legacy/test_sca/test_basic_usage/data/sca_files_test_suite.yml:153]: Done, thank you!

**Pull request merged**